### ISSUE TITLE
Export keras model

### ISSUE THREAD

**Issue by Nesty92** [keras-team/autokeras#1027]
### Bug Description
```python
Traceback (most recent call last):
  File "/home/nesty/PycharmProjects/autokeras/imdb.py", line 54, in <module>
    model.save('model.h5')
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/engine/network.py", line 1008, in save
    signatures, options)
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/saving/save.py", line 112, in save_model
    model, filepath, overwrite, include_optimizer)
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/saving/hdf5_format.py", line 109, in save_model_to_hdf5
    save_weights_to_hdf5_group(model_weights_group, model_layers)
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/saving/hdf5_format.py", line 627, in save_weights_to_hdf5_group
    weight_values = K.batch_get_value(weights)
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/backend.py", line 3270, in batch_get_value
    return [x.numpy() for x in tensors]
  File "/home/nesty/PycharmProjects/autokeras/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/backend.py", line 3270, in <listcomp>
    return [x.numpy() for x in tensors]
AttributeError: 'TrackableWeightHandler' object has no attribute 'numpy'
```

### Bug Reproduction
Code for reproducing the bug:
```python
import numpy as np
import tensorflow as tf

import autokeras as ak

def imdb_raw():
    max_features = 20000
    index_offset = 3  # word index offset

    (x_train, y_train), (x_test, y_test) = tf.keras.datasets.imdb.load_data(
        num_words=max_features,
        index_from=index_offset)
    x_train = x_train
    y_train = y_train.reshape(-1, 1)
    x_test = x_test
    y_test = y_test.reshape(-1, 1)

    word_to_id = tf.keras.datasets.imdb.get_word_index()
    word_to_id = {k: (v + index_offset) for k, v in word_to_id.items()}
    word_to_id["<PAD>"] = 0
    word_to_id["<START>"] = 1
    word_to_id["<UNK>"] = 2

    id_to_word = {value: key for key, value in word_to_id.items()}
    x_train = list(map(lambda sentence: ' '.join(
        id_to_word[i] for i in sentence), x_train))
    x_test = list(map(lambda sentence: ' '.join(
        id_to_word[i] for i in sentence), x_test))
    x_train = np.array(x_train, dtype=np.str)
    x_test = np.array(x_test, dtype=np.str)
    return (x_train, y_train), (x_test, y_test)


(x_train, y_train), (x_test, y_test) = imdb_raw()
print(x_train.shape)  # (25000,)
print(y_train.shape)  # (25000, 1)
print(x_train[0][:50])  # <START> this film was just brilliant casting <UNK>

clf = ak.TextClassifier(max_trials=10)
# Search for the best model.
clf.fit(x_train, y_train, epochs=2)
# Evaluate on the testing data.
print('Accuracy: {accuracy}'.format(accuracy=clf.evaluate(x_test, y_test)))

model = clf.export_model()
model.save('model.h5')
```
Data used by the code:
https://keras.io/datasets/#imdb-movie-reviews-sentiment-classification
### Expected Behavior
export the model

### Setup Details
Include the details about the versions of:
 - OS type and version: LinuxMint 19.3 (ubuntu 18.04)
 - Python: 3.6
 - autokeras: 1.0.2
 - keras-tuner:1.0.1
 - scikit-learn:0.22.2
 - numpy: 1.18.1
 - pandas: 1.0.1
 - tensorflow: 2.1.0

**Comment by Nesty92** [comment]
Related with #999

**Comment by haifeng-jin** [comment]
This bug is fixed with the latest release 1.0.5.
